### DISCUSSION TITLE
Prefect configuration in config.toml seems not working properly!

### DISCUSSION THREAD

**Discussion by den-pro-soft** [PrefectHQ/prefect#3580] [General]
# Description
Hello!
Please look at the attached screenshot. I am deploying the Prefect Server on the EC2 instance. I changed the address for server.ui.graphql_url in config.toml to reflect this. However this isn't reflected in UI, which keeps referring to localhost.

![image](https://user-images.githubusercontent.com/34548664/97425220-0a989200-18cf-11eb-891d-ac9d90325ded.png)

# Result
The localhost is changed to the ip adress

# Deployment Step I did
1. Used the following config.toml file:
```
[server]
host = "http://34.209.184.154"
port = "4200"
host_port = "4200"
endpoint = "${server.host}:${server.port}"

        [server.graphql]
        host = "34.209.184.154"

        [server.ui]
        host = "http://34.209.184.154"
        port = "8080"
        host_port = "8080"
        apollo_url = "http://34.209.184.154:4200/graphql"
```
2. docker system prune -a ( optional )
3. prefect server start
4. http://34.209.184.154:8080/

Could you please let me know what I'm missing? 
Looking forward to hearing from you!
Thanks.

**Comment by cicdw** [comment]
Duplicate of https://github.com/PrefectHQ/server/issues/119
